### PR TITLE
Fix hamburger menu

### PR DESCRIPTION
**Pull request by bazelajoanna**
Fix menu transition and appearance.

### CODE REVIEW

File: src/components/atoms/Navbar/index.tsx [hunk 1]
```diff
@@ -33,24 +33,26 @@ const Navbar: FC = () => {
...
-          className={navbarMenuStyle}
-          onClick={toggleHamburger}
-          src={isNavbarMenuActive ? hamburgerX : hamburger}
-          aria-hidden
-          alt="Ikona menu"
-        ></S.HamburgerImg>
-      </S.MenuIconButton>
-      <S.NavbarMenu>
-        <S.NavbarMenuList className={navbarMenuStyle}>
+    <S.HeaderWrapper>
```
Comment (franiakat) [inline src/components/atoms/Navbar/index.tsx:36]: I think this is an `organism` already.

File: src/components/atoms/Navbar/styles.tsx [hunk 2]
```diff
@@ -1,76 +1,102 @@
...
-import { minDevice } from "../../../assets/styles/breakpoints";
-import { activeLink } from "../../../assets/styles/colors";
 import Wrapper from "../Wrapper";
 
-const NavbarDiv = styled(Wrapper)`
-  height: 8rem;
+import { minDevice } from "../../../assets/styles/breakpoints";
+import { activeLink, white } from "../../../assets/styles/colors";
+
+const HeaderWrapper = styled.div`
```
Comment (franiakat) [inline src/components/atoms/Navbar/styles.tsx]: I think we should have `header` tag here instead.

File: src/components/atoms/Navbar/styles.tsx [hunk 3]
```diff
@@ -1,76 +1,102 @@
...
+  position: relative;
+  z-index: 11;
+  display: flex;
+`;
+
+const NavbarWrapper = styled(Wrapper)`
+  height: 8rem;
+  width: 100%;
+  position: relative;
+  z-index: 12;
```
Comment (franiakat) [inline src/components/atoms/Navbar/styles.tsx:20]: I think this isn't needed.

File: src/components/atoms/Navbar/styles.tsx [hunk 4]
```diff
@@ -1,76 +1,102 @@
...
+  }
+`;
+
+const NavbarMenuList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: flex-start;
+  padding-left: 2rem;
+
```
Comment (evulins) [inline src/components/atoms/Navbar/styles.tsx:83]: We could add it here width: 100vw to prevent strange behavior while it closing

**Review by franiakat — COMMENTED** [review]
Tiny things in comments, awesome cleanup and fix 🧚

File: src/components/atoms/Wrapper/index.tsx [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 
-const Wrapper = styled.div<PropsOptional>`
+const Wrapper = styled.div<WrapperProps>`
   display: flex;
   flex-direction: ${(props) => props.flexDirection || "row"};
   justify-content: ${(props) => props.justifyContent || "inherit"};
   align-items: ${(props) => props.alignItems || "inherit"};
-  padding: ${(props) => props.padding || "1.6rem"};
-  margin: ${(props) => props.margin || "1.6rem"};
+  padding: ${(props) => props.padding || "0 1.6rem"};
```
Comment (evulins) [inline src/components/atoms/Wrapper/index.tsx:11]: Have you checked if this change in padding doesn't mess with styles in other places?

**Review by evulins — COMMENTED** [review]
Great job! Looks great! Some small questions :)

File: src/components/atoms/Navbar/styles.tsx (hunk 3 above)
Comment (bazelajoanna) [reply src/components/atoms/Navbar/styles.tsx:20]: I've removed height and position, but width and z-index are necessary here.

File: src/components/atoms/Navbar/styles.tsx (hunk 4 above)
Comment (bazelajoanna) [reply src/components/atoms/Navbar/styles.tsx:83]: Good suggestion, thanks! :)

Comment (evulins) [reply src/components/atoms/Navbar/styles.tsx:83]: But only for mobile devices with hamburger :D

Comment (franiakat) [reply src/components/atoms/Navbar/styles.tsx:83]: 🍔

File: src/components/atoms/Navbar/styles.tsx (hunk 3 above)
Comment (franiakat) [reply src/components/atoms/Navbar/styles.tsx:20]: Can we have a quick call? I've removed it locally and was working well for me, maybe I missed something ;)

File: src/components/atoms/Navbar/styles.tsx (hunk 4 above)
Comment (bazelajoanna) [reply src/components/atoms/Navbar/styles.tsx:83]: Yes, I've noticed that 👩‍🎓

File: src/components/atoms/Wrapper/index.tsx (hunk 5 above)
Comment (bazelajoanna) [reply src/components/atoms/Wrapper/index.tsx:11]: Yes, everything works :) I've just changed the line height in FooterP and now it looks even better 🌷

Comment (evulins) [reply src/components/atoms/Wrapper/index.tsx:11]: Wohooooo 🍾

**Review by evulins — CHANGES_REQUESTED** [review]
👍

**Review by evulins — APPROVED** [review]

**Review by franiakat — APPROVED** [review]
sqash

**Pull request merged**